SPECS/reaper/reaper.spec

```diff
@@ -37,6 +37,7 @@ Patch1:         CVE-2023-42282.patch
 Patch2:         CVE-2017-18214.patch
 Patch3:         CVE-2024-42459.patch
 Patch4:         CVE-2024-43796.patch
+Patch5:         CVE-2024-45590.patch
```

Comment:
NACK.
I feel that we have `autosetup -N` which disable automatic patching.
We might want to take a closer look at it.

Reply:
yes, that's right but line 97 has autopatch